branding/pdf/entities.adoc

```diff
@@ -47,6 +47,8 @@
 :sle: SUSE Linux Enterprise
 :sleses: SUSE Linux Enterprise Server with Expanded Support
 :es: Expanded Support
+:slemicro: SLE Micro
+:microos: MicroOS
```

Comment:
I guess you mean `openSUSE MicroOS` here?

Reply:
yeah, I was going to use the `{opensuse}` entity before it.